airbyte-config/config-models/src/main/resources/types/StandardConnectorBuilderReadOutput.yaml

```diff
@@ -0,0 +1,12 @@
+---
+"$schema": http://json-schema.org/draft-07/schema#
+"$id": https://github.com/airbytehq/airbyte/blob/master/airbyte-config/models/src/main/resources/types/StandardCheckConnectionOutput.yaml
+title: StandardConnectorBuilderReadOutput
+description: describes
+type: object
+required:
+  - output
+additionalProperties: false
+properties:
+  output:
```

Comment:
is this the request response string?

Reply:
this is actually just an arbitrary output. I didn't create the real models in this PR.